cmd/agent/main.go

```diff
@@ -160,6 +163,7 @@ func run() error {
 			},
 			nettestContainerImage: *cagentNettestContainerImage,
 		}
+		log.Debugf("Agent started using MonitoringConfig %#v", monitoringConfig)
```

Comment:
I think this is the 3rd time I see this config being logged. Do you really need this? :)

Reply:
No I was trying to understand where was the configuration lost and added similar debugging statements in multiple places.